pom.xml

```diff
@@ -34,8 +34,8 @@
   </developers>
 
   <properties>
-    <maven.compiler.source>1.6</maven.compiler.source>
-    <maven.compiler.target>1.6</maven.compiler.target>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
```

Comment:
Why the java version bump?

Reply:
Try with resources in a test. I suppose that does not merit the change. I'll revert these.